services/uniauth/login_check.py

```diff
@@ -0,0 +1,87 @@
+from . import login_db
+from . import token_db
+
+def _uname_check(username: str):
+    if len(username) < 6 or len(username) > 20:
```

Comment:
```suggestion
    if (len(username) < 6 or len(username) > 20) and username != "cxzlw":
```

Add an 例外 to allow my username :)